test/opt/local_access_chain_convert_test.cpp

```diff
@@ -69,6 +69,34 @@ OpName %gl_FragColor "gl_FragColor"
 %gl_FragColor = OpVariable %_ptr_Output_v4float Output
 )";
 
+  const std::string predefs_after =
```

Comment:
I see that %int_1 = OpConstant %int 1 is now removed by the pass.  Presumably that's because this pass is now using a more generalized set of utilities.  Ok by me.